src/SFA.DAS.ApprenticeFeedback.Jobs/SFA.DAS.ApprenticeFeedback.Jobs.csproj

```diff
@@ -24,13 +24,14 @@
     <PackageReference Include="RestEase" Version="1.5.5" />
     <PackageReference Include="RestEase.HttpClientFactory" Version="1.5.5" />
     <PackageReference Include="SFA.DAS.ApprenticeCommitments.Messages" Version="0.1.83-prerelease-6" />
+    <PackageReference Include="SFA.DAS.ApprenticeFeedback.Messages" Version="0.1.21" />
     <PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="3.0.84" />
     <PackageReference Include="SFA.DAS.Http" Version="3.2.65" />
     <PackageReference Include="SFA.DAS.NLog.Targets.Redis" Version="1.2.1" />
   </ItemGroup>
   <ItemGroup>
     <None Update="local.settings.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Unless there's a specific reason I think it's safest to set this to Always. File dates can have unexpected values for various reasons

Reply:
Reverted